proto/expression.proto

```diff
@@ -165,13 +165,27 @@ enum ExprType {
 	Like = 4005;
 	RLike = 4006;
 	Case = 4007;
+
+	/* Scalar Function */
+	ScalarFunc = 10000;
+}
+
+enum ScalarFuncSig {
+	/* Math Arithmatic */
+	PlusInt = 1;
+	PlusReal = 2;
+	PlusString = 3;
+	PlusDecimal = 4;
+	PlusTime = 5;
+	PlusDuration = 6;
 }
 
 // Evaluators should implement evaluation functions for every expression type.
 message Expr {
 	optional ExprType tp = 1 [(gogoproto.nullable) = false];
 	optional bytes val = 2;
 	repeated Expr children = 3;
+	optional ScalarFuncSig sig = 4;
```

Comment:
@hhkbp2 
Sorry for late reply,
so we still use `ExprType tp` to store the function class name 
or
define a new `ExprType` like `ExprType_ScalarFunction` to indicate the scalarFunction PBs